gatsby-node.js

```diff
@@ -380,36 +383,60 @@ exports.onCreatePage = ({ page, actions }) => {
     }
 }
 
-const StylelintPlugin = require('stylelint-webpack-plugin')
-const TerserPlugin = require('terser-webpack-plugin')
-
 const style_lint_options = {
     files: 'src/**/*.js',
     emitErrors: false,
     lintDirtyModulesOnly: true,
 }
 
-exports.onCreateWebpackConfig = ({ stage, actions, loaders, getConfig }, { ...options }) => {
-    const config = getConfig()
-    if (config.optimization) {
-        config.optimization.minimizer = [new TerserPlugin()]
-    }
-    if (stage === 'build-html' || stage === 'develop-html') {
-        actions.setWebpackConfig({
-            module: {
-                rules: [
-                    {
-                        test: /analytics/,
-                        use: loaders.null(),
-                    },
-                ],
-            },
-        })
-    }
+exports.onCreateWebpackConfig = ({ stage, actions, loaders }, { ...options }) => {
+    // const config = getConfig()
+    // const isProduction = config.mode === 'production'
+    const isProduction = true
+
     actions.setWebpackConfig({
-        plugins: [new StylelintPlugin({ ...style_lint_options, ...options })],
+        devtool: false, // enable/disable source-maps
```

Comment:
Should be `devtool: !isProduction`

Reply:
<img width="1188" alt="Screenshot 2024-01-22 at 5 37 54 PM" src="https://github.com/binary-com/deriv-com/assets/107860964/2683ef8a-1199-4827-b643-425258177317">

Reply:
i'll change with ternary operator